google-account-plugin/src/com/google/gct/idea/AccountPluginInitializationComponent.java

```diff
@@ -63,13 +53,13 @@ void enableFeedbackUtil() {
    * and is therefore not subscribed and listening.
    */
   @VisibleForTesting
-  void enableUsageTracking() {
+  void maybeEnableUsageTracking() {
```

Comment:
changed it to 'configureUsageTracking' which is less specific about whether it gets turned on or off.